rfcs/slider-component.md

```diff
@@ -0,0 +1,113 @@
+# Slider Component
+
+Represents control with sliding axis to choose a single value or range between min and max values.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {Slider} from 'baseui/slider';
+
+export default () => <Slider range={[
+  {value: 0, label: '0, min'},
+  {value: 100, label: '100, max'}
+]} />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {Slider} from 'baseui/slider';
```

Comment:
This should be StatefulSlider if you're using `initialState` below, right?

Reply:
correct